spec/acceptance/self_service_spec.rb

```diff
@@ -149,6 +149,14 @@ class {'self_service':
         expect(result.stdout).to match(%r{false})
         run_shell('puppet config set use_cached_catalog false', expect_failures: false)
       end
+
+      it 'if S0034 conditions for false are met' do
+        run_shell('mv /var/log/puppetlabs/installer/*.install.log  /var/log/puppetlabs/installer/install_log.orig ; touch -d "2 years ago" 2022-01-06T09.45.21+0000.install.log')
```

Comment:
```suggestion
        run_shell('mv /var/log/puppetlabs/installer/*.install.log  /var/log/puppetlabs/installer/install_log.orig ; touch -d "2 years ago" /var/log/puppetlabs/installer/2022-01-06T09.45.21+0000.install.log')
```

Reply:
Actually, maybe we can just send the timestamp of the file back 2 years, run the test, and then update it.

```
        run_shell('touch -d "2 years ago" /var/log/puppetlabs/installer/*.install.log')
        result = run_shell('facter -p self_service.S0034')
        expect(result.stdout).to match(%r{false})
        run_shell('touch /var/log/puppetlabs/installer/*.install.log')
```